bin/stacks/routing-database-stack.ts

```diff
@@ -36,6 +36,11 @@ export const DynamoDBTableProps = {
     PartitionKeyName: 'cacheKey',
     SortKeyName: 'block',
   },
+  TokenValidationResultDbTable: {
+    Name: 'TokenValidationResultDB',
+    PartitionKeyName: 'tokenAddress',
+    SortKeyName: 'chainId',
+  },
```

Comment:
I wonder if we should have:
```
paritionKey: chainIdTokenAddress,
sortKey: blockNumber
```

And we also do something like only checking every several blocknumbers?

WDYT?

Reply:
So the way I'm thinking is that if the token is FOT/SFT, most likely they will remain that way beyond couple blocks right? So I was thinking to have a longer TTL for that. Do you see any benefit of checking every several blocknumbers, that having a longer TTL might miss?

Reply:
given tht this might be used for other properties too, I think Miguel's suggestion is a good one!

Reply:
I adopted the feedback. Due to github issue, the PR hasn't reflected the changes yet.

Reply:
Looks like PR reflected my commits. Ready for review again.